kinetic/distribution.yaml

```diff
@@ -1569,6 +1569,24 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/dynamixel-workbench.git
       version: kinetic-devel
     status: developed
+  dynamixel-workbench-msgs:
+    doc:
+      type: git
+      url: https://github.com/ROBOTIS-GIT/dynamixel-workbench-msgs.git
+      version: kinetic-devel
+    release:
+      packages:
+      - dynamixel_workbench_msgs
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ROBOTIS-GIT-release/dynamixel-workbench-msgs-release.git
+      version: 0.1.5-1
+    source:
+      test_pull_requests: true
```

Comment:
When you ran `bloom` it asked you about enabling this flag. Please re-read the [message] (https://github.com/ros-infrastructure/bloom/blob/d48aa0277639952e250706b54f1d3bbc786c395e/bloom/commands/release.py#L533) which also referenced some docs to describe the process further.